src/ray/raylet/scheduling/cluster_resource_manager.cc

```diff
@@ -89,6 +89,12 @@ bool ClusterResourceManager::UpdateNode(scheduling::NodeID node_id,
     local_view.object_pulls_queued = resource_data.object_pulls_queued();
   }
 
+  // Update the idle duration for the node in terms of resources usage.
```

Comment:
Looks like the major change is we don't run the following code when 

resources_available_changed = false in other method. Is there any implication in this change? Now it means although available is not changed, we can update the nodes (is it to accomodate the case where new resources are created)?

Reply:
Actually after ray syncer, resources_available_changed is always true since we don't have the old lightweight resource report. cc @iycheng correct me if I'm wrong.

Reply:
Ok, completely removed the `resources_available_changed` since it was only used by lightweight resource report.

Reply:
should we just assert on resources_available_changed ?

Reply:
Discussed with Yi, we think we can just remove it Completely